### ISSUE TITLE
Apache Commons

### ISSUE THREAD

**Issue by rdifrango** [gaul/modernizer-maven-plugin#21]
Should there be provisions in the configuration to look for usages from Apache Commons Collections, like functions, closures, etc.?

**Comment by andrewgaul** [comment]
@rdifrango Definitely!  Could you submit a pull request for more Apache Commons checks?  I only have a little familiarity with commons-collections since I mostly use Guava.

**Comment by rdifrango** [comment]
@andrewgaul We use it sparingly as we've started to switch over to Guava, so I'll give it a shot.

**Comment by andrewgaul** [comment]
@rdifrango Switching to Guava is another interesting use case -- Modernizer could provide a second optional resource file to suggest Commons replacements.  Modernizer only supports a single resource file now but supporting more is easy to add.

**Comment by Stephan202** [comment]
+1 for the idea of generalizing the plugin to facilitate migrations from old lib _X_ to newer lib _Y_ (where in this case _X_ = Apache Commons and _Y_ is Guava). I guess _X_ and _Y_ could even be different versions of the same library. (Though that's a less useful scenario, provided the library maintainers mark methods with a more suitable replacement `@Deprecated`.)

**Comment by andrewgaul** [comment]
For example, suggest replacing `org.apache.commons.io.input.BoundedInputStream` with `com.google.common.io.ByteStreams.limit`
